README.md

```diff
@@ -1,32 +1,16 @@
-# terraform-ap-modules
```

Comment:
Pls don't delete the main repo README

Reply:
shall I change them back? This file is generated by terraform-doc